dotnet/src/VectorDataIntegrationTests/SqlServerIntegrationTests/DependencyInjection/SqlServerDependencyInjectionTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.SemanticKernel.Connectors.SqlServer;
+using VectorDataSpecificationTests.DependencyInjection;
+using VectorDataSpecificationTests.Models;
+using Xunit;
+
+namespace SqlServerIntegrationTests.DependencyInjection;
+
+public class SqlServerDependencyInjectionTests : DependencyInjectionTests<string, SimpleRecord<string>>
```

Comment:
Note that we can have general tests for this, where each connector exposes a list of its delegates which register in DI, and then confirm that they do what they should be.